### PR TITLE
Included StoneCo

### PR DESCRIPTION
**Pull request by renanmarcos**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [x] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
Hi @renanmarcos, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 712f2b5).  See below for details:

```
stone-co.md: Company title "StoneCo" doesn't match filename (expected ~ "stoneco.md")

1 problem detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by auto-nylen** [comment]
Hi @renanmarcos, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit f4574ea). :heavy_check_mark:

File: company-profiles/stoneco.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Our purpose is clear: changing the way the payment system works. To do so, we work hard to provide an equal playing field for entrepreneurs.
+
+## Company size
+
+4000+ employees and constantly growing to innovate and bring solutions to our reason: the customer.
+
+## Remote status
+
+For many teams we offer full remote options, but you can work either in offices around Brazil as partially remote depending from your team and position.
```
Comment (nylen) [inline company-profiles/stoneco.md:15]: >you can work either in offices around Brazil as partially remote depending from your team and position.
This isn't very clear and there are a few errors with prepositions.
Maybe rephrase like this?
```suggestion
For many teams we offer fully remote options, and for the rest you can work either in offices around Brazil or partially remote depending on your team and position.
```

File: company-profiles/stoneco.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Region
+
+Brazil
+
+## Company technologies
+
+Tools and technologies at StoneCo generally is decided by each squad and varies according the solution. Depending from your squad, you can work with: C#, Elixir, Kubernetes, Docker, RabbitMQ, GCP, Azure, Ansible, Terraform and much more. You can check the job description to know more about a specific stack.
```
Comment (nylen) [inline company-profiles/stoneco.md:23]: Grammar fixes:
```suggestion
Tools and technologies at StoneCo generally are decided by each squad and vary according to the solutions needed. Depending on your squad, you can work with: C#, Elixir, Kubernetes, Docker, RabbitMQ, GCP, Azure, Ansible, Terraform and many more. You can check the job description to know more about a specific stack.
```

File: company-profiles/stoneco.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Office locations
+
+The main offices is located at São Paulo and Rio de Janeiro, but we has more around Brazil.
```
Comment (nylen) [inline company-profiles/stoneco.md:27]: Grammar fixes:
```suggestion
The main offices are located in São Paulo and Rio de Janeiro, but we have more around Brazil.
```

File: company-profiles/stoneco.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Company blurb
+
+[StoneCo](https://www.stone.co/) is a leading provider of financial technology solutions that empower merchants and integrated partners to conduct electronic commerce seamlessly across in-store, online, and mobile channels. Stone has developed a deep client-centric culture that seeks to delight our clients rather than to simply provide them with a solution or service. Stone serves clients of all sizes and types that transact online, offline or have an omni-channel sales approach. Stone also serves many integrated partners, which use or embed Stone’s solutions into their own offerings to enable their customers to conduct commerce more conveniently.
+
+Our purpose is clear: changing the way the payment system works. To do so, we work hard to provide an equal playing field for entrepreneurs.
+
+## Company size
+
+4000+ employees and constantly growing to innovate and bring solutions to our reason: the customer.
```
Comment (nylen) [inline company-profiles/stoneco.md:11]: More clear since a "reason" is always a reason "for" something:
```suggestion
4000+ employees and constantly growing to innovate and bring solutions to our reason for existing: the customer.
```

**Comment by nylen** [comment]
Olá @renanmarcos! I made a few suggestions for grammar fixes but other than that your PR looks good.

**Comment by renanmarcos** [comment]
@nylen Olá! I've learned a lot from your review, thank you. 😄
